schema/common/defaults/configuration/logging.yml

```diff
@@ -2,7 +2,10 @@ kind: configuration/logging
 title: Logging Config
 name: default
 specification:
-  cluster_name: EpiphanyElastic
+  cluster_name: EpiphanyOpenSearch
+  opensearch_user: opensearch
```

Comment:
Specify in variable that this is an os user? Maybe also separate group?

Reply:
ad a)
ok
Implemented now.
ad b)
As discussed before: there are two approaches and it's hard to say which is better.
Probably would be good to discuss it within the team before changing.